tests/unit/PaymentErrorDialog.spec.ts

```diff
@@ -138,4 +138,81 @@ describe('Payment Error Dialog', () => {
 
     wrapper.destroy()
   })
+
+  it('renders PAD error messages correctly when they are present', () => {
```

Comment:
PAD = pre-authorized debit (another way to pay that is not CC or BCOL)

It doesn't look like these 2 tests are PAD-specific so I'd remove that from their title.

Reply:
Sure. I'll remove it from the title.